### ISSUE TITLE
If a custom shape of node changes due a movement, has to be able to undo/redo

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#3326]
In an edge of node, or the node itself, whith a custom shape (for example, after a recomputing) is moved, the old shape must be saved using GNEChange_Parameter, and undo/redo with ctrl+Z/Y

**Comment by namdre** [comment]
@palvarezlopez Is this still an issue?

**Comment by palvarezlopez** [comment]
@namdre Yes. I tried to repair it, but it's a little bit complicated. Lets do it for the 1.1.0
